torch/csrc/jit/tracer.cpp

```diff
@@ -437,8 +437,36 @@ void addInputs(Node* n, const char* name, at::IntArrayRef value) {
 void addInputs(Node* n, const char* name, const ArrayRef<double>& value) {
   AT_ERROR("Tracing float lists currently not supported!");
 }
-void addInputs(Node* n, const char* name, const std::vector<double>& value) {
-  AT_ERROR("Tracing float lists currently not supported!");
+
+void addInputs(
```

Comment:
what about addInputs above that takes ArrayRef. Since ArrayRef is constructible from vector, we probably need to keep only one

Reply:
Now vector one call the arrayref one